CharlieBackend.Api/Controllers/ThemesController.cs

```diff
@@ -36,5 +34,24 @@ public async Task<ActionResult<List<ThemeModel>>> GetAllThemes()
                 return StatusCode(500); 
             }
         }
+
+        [Authorize(Roles = "2, 4")]
+        [HttpPost]
+        public async Task<ActionResult<IList<ThemeModel>>> PostThemes([FromBody] ThemeModel addThemeModel)
+        {
+            try
+            {
+                await _themeService.CreateThemeAsync(addThemeModel);
+
+                return Ok(addThemeModel);
+
+            }
+            catch
+            {
+                return StatusCode(500);
+            }
```

Comment:
Please remove try catch, because CreateThemeAsync already catches all exceptions. Even if it would throw unhandled exception ASP.NET core will set response status code to 500.

Reply:
Done. Please review it